docs/docs/tutorials/real-world-example-with-react/15-social-auth.md

```diff
@@ -175,12 +175,12 @@ subControllers = [
   controller('/stories', StoriesController),
   controller('/auth', AuthController),
   controller('/profile', ProfileController),
-  controller('/auth', SocialAuthController)
+  controller('/oauth', SocialAuthController)
```

Comment:
The React application sends the request to `/auth/google`. The path is correct here and changing it to `/oauth` will break the application.